exonum-java-binding/core/rust/ejb-app/src/node_builder.rs

```diff
@@ -17,16 +17,80 @@
 use exonum_btc_anchoring::ServiceFactory as BtcAnchoringServiceFactory;
 use exonum_configuration::ServiceFactory as ConfigurationServiceFactory;
 use exonum_time::TimeServiceFactory;
-use java_bindings::exonum::helpers::fabric::{self, ServiceFactory};
-use java_bindings::utils::{
-    load_enabled_services, BTC_ANCHORING_SERVICE, CONFIGURATION_SERVICE, EJB_SERVICE,
-    PATH_TO_SERVICES_TO_ENABLE, TIME_SERVICE,
+use java_bindings::{
+    exonum::helpers::fabric::{self, ServiceFactory},
+    utils::services::{
+        load_services_definition, EjbAppServices, BTC_ANCHORING_SERVICE, CONFIGURATION_SERVICE,
+        PATH_TO_SERVICES_DEFINITION, TIME_SERVICE,
+    },
+    JavaServiceFactoryAdapter,
 };
-use java_bindings::JavaServiceFactory;
 
-use std::collections::{HashMap, HashSet};
-use std::path::Path;
+use std::{
+    collections::{HashMap, HashSet},
+    path::Path,
+};
+
+/// Creates `NodeBuilder` using services configuration from `ejb_app_services.toml` located in the working directory.
+pub fn create() -> fabric::NodeBuilder {
+    let service_factories = prepare_service_factories(PATH_TO_SERVICES_DEFINITION);
+    let mut builder = fabric::NodeBuilder::new();
+    for service_factory in service_factories {
+        builder = builder.with_service(service_factory);
+    }
+    builder
+}
+
+// Prepares vector of `ServiceFactory` from services configuration file located at given path.
+// Panics in case of problems with reading/validating of service configuration file.
```

Comment:
Will there be a *proper* user-friendly error message in case this code panics (**cc** @dip56 )? 

Shall panics be used in such a case at all?

Reply:
The idea is because this piece of code is used from the top-level code of the console app panics are pretty OK. We panic because we can't populate builder with required data.

Reply:
> panics are pretty OK

@vitvakatu Do you agree with this statement?

How the output looks like?